### ISSUE TITLE
Interaction with Noia-Terminal

### ISSUE THREAD

**Issue by Pierozi** [noia-network/noia-node-gui#15]
It is possible to connect the gui to existing noia-node started through `noia-terminal` for instance ?

Because actually the gui try to start a newer node.

**Comment by Dancia** [comment]
Currently there is no such functionality to connect GUI straight to node running in terminal mode, since node core is build-in inside GUI package.

At this moment either you should pick GUI or terminal node. The underlying core which handles node logic https://github.com/noia-network/noia-node is the same on both options. GUI package just has more graphics interface fixes.

I will mark this issue as an enhancement, since in future GUI could actually be separated so it can be connected to any headless node when given correct authentication.

**Comment by Pierozi** [comment]
Yes, I understand, this feature would require some design change regarding the communication with the node by a secure channel.

Maybe could be better to improve a bit the `noia-node-terminal` to add command for have the same information as GUI, the number of minutes, bandwidth.

**Comment by Dancia** [comment]
Yes, you are correct. It would require some decision on how to implement the secure channel and redesing coomuncation. Currently the front-end is built with angular. Angular communicated with electron process via IPC (interprocess communication). To make it more flexible (de-attach front-end), either communication should be redesigned to use HTTP RESTful API, WebSockets or some other secure channel.

It is open to discussion.

**Comment by hamidrezaghorbani** [comment]
I love your project.
